### PR TITLE
Added Documentation for Top View of Binary Tree 

### PR DESCRIPTION
**Pull request by Saatvik21**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

`Yes`

### Description

I've added an article for top view of binary tree.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes : #470

### CODE REVIEW

File: docs/data-structures/depth-of-tree.mdx [hunk 1]
```diff
@@ -267,4 +267,4 @@ Depth of tree : 3
 ## Credits
 
 - [Satyam Dengre](https://github.com/satyampgt4) for the [C++ implementation](https://github.com/TesseractCoding/NeoAlgo/blob/master/C-Plus-Plus/graphs/Depth_of_tree.cpp)
-- [Sangeeta Mishra](https://github.com/SangeetaMishra) for the [Python implementation](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/graphs/Depth_Of_Tree.py)
\ No newline at end of file
+- [Sangeeta Mishra](https://github.com/SangeetaMishra) for the [Python implementation](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/graphs/Depth_Of_Tree.py)
```
Comment (gat786) [inline docs/data-structures/depth-of-tree.mdx:270]: Why is this file commited?.

File: docs/competitive-programming/top-view-of-BT.mdx [hunk 2]
```diff
@@ -0,0 +1,204 @@
+---
```
Comment (gat786) [inline docs/competitive-programming/top-view-of-BT.mdx:1]: Just a suggestion please try to keep the width of each line in the file to be at most 79 characters. It makes the content center heavy and people won't have to scroll left or right to see what you have written.

**Review by gat786 — CHANGES_REQUESTED** [review]

**Comment by gat786** [comment]
Everything looks great just make sure you look on the comments I have made.

File: docs/data-structures/depth-of-tree.mdx (hunk 1 above)
Comment (Saatvik21) [reply docs/data-structures/depth-of-tree.mdx:270]: It got changed by itself when I tried to rebase, but I don't understand the change in both of them. They look similar.

File: docs/competitive-programming/top-view-of-BT.mdx (hunk 2 above)
Comment (Saatvik21) [reply docs/competitive-programming/top-view-of-BT.mdx:1]: I don't get what you mean. Can you elaborate?

Comment (gat786) [reply docs/competitive-programming/top-view-of-BT.mdx:1]: Like each line should contain up to 79 characters only.

Comment (Saatvik21) [reply docs/competitive-programming/top-view-of-BT.mdx:1]: Alright, I'll make sure that happens in the future.

File: docs/data-structures/depth-of-tree.mdx (hunk 1 above)
Comment (Saatvik21) [reply docs/data-structures/depth-of-tree.mdx]: @gat786 I've removed this commit now.

Kindly review :)

Comment (gat786) [reply docs/data-structures/depth-of-tree.mdx]: can you unstage this file?

Comment (Saatvik21) [reply docs/data-structures/depth-of-tree.mdx]: It's unstaged now, you can look at "files changed" and it doesn't appear now.

**Comment by Saatvik21** [comment]
@gat786 can you also add labels to this PR?

**Pull request merged**